.pre-commit-config.yaml

```diff
@@ -4,18 +4,12 @@ ci:
 
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.1.1"
+    rev: "v0.1.3"
     hooks:
       - id: ruff
+        types_or: [python, pyi, jupyter]
         args: [--fix, --ignore=E741, --exclude=__init__.py]
```

Comment:
Can you also move these configs to `ruff.toml`?

Reply:
how can i do that ?

if using [docs](https://docs.astral.sh/ruff/settings/#extend-include)  i add this to ruff.toml it not works 

![image](https://github.com/pybamm-team/PyBaMM/assets/113838908/13ad3850-fc2a-4d65-a433-4dcf3aec6b3f)

Reply:
The documentation for `ruff.toml` (https://docs.astral.sh/ruff/configuration/#using-rufftoml) mentions "though in the ruff.toml and .ruff.toml versions, the [tool.ruff] header is omitted", which should help

Reply:
test run smoothly but i don't have any effect on code even after i remove `E703` from list of ignore.

it means after removing 10th line from pre-commit-config.yaml and improving  ruff.toml  is not working.

`ruff.toml` like this  and with removed `10th line in pre-commit-config.yaml`

![image](https://github.com/pybamm-team/PyBaMM/assets/113838908/28767709-375f-4bfe-a503-9dbb27553d0a)

Reply:
@Rjchauhan18, it would be easier for us to help you if you push your changes (don't worry about the CI failing once you push it - we can help you fix the CI, but it becomes challenging to help you without looking at your entire changes).

Also, avoid adding screenshots of your logs; instead, copy-paste them into markdown (good practice).